apps/web/lib/__experimental__/replicache-context.tsx

```diff
@@ -25,7 +26,8 @@ const hoistReplicache = (
   if (currentWindow.parent === currentWindow.self) {
     currentWindow.replicache = new Replicache({
       // See https://doc.replicache.dev/licensing for how to get a license key.
-      licenseKey: process.env.NEXT_PUBLIC_REPLICACHE_LICENSE_KEY!,
+      licenseKey: config.next.public.replicache.licenceKey!,
```

Comment:
Can we do
```suggestion
      licenseKey: config.next.public.replicache?.licenceKey,
```
instead?

Reply:
okay, doing that. 

If we're not 100% sure about having the specific object key, we shouldn't use `!`